src/main/java/frc/robot/subsystems/NavXSubsystem.java

```diff
@@ -31,5 +32,16 @@ public void initialize() {
             ;
```

Comment:
see line of code--this is an infinte loop with  no timeout-- BAD! if the navx is unplugged, the robot code will hang at this. At a minimimum, in this loop we should print 'navx initializing' so that we know we're stuck there. better would be to give a timeout somehow.